rules/windows/process_creation/proc_creation_win_wsl_windows_binaries_execution.yml

```diff
@@ -0,0 +1,23 @@
+title: Windows Binary Executed From WSL
+id: ed825c86-c009-4014-b413-b76003e33d35
+status: experimental
+description: Detects execution of windows binaries from wihthin a WSL instance. This could be used to avoid parent/child relationship detections or similar
```

Comment:
```suggestion
description: Detects the execution of Windows binaries from within a WSL instance. This could be used to masquerade parent-child relationships
```